index.css

```diff
@@ -38,8 +44,14 @@ p {
 .status strong:before {
 	content: url(logo.svg) "\A";
 	white-space: pre;
+	filter: invert(100%);
 }
 input {
+	background-color: var(--input-box-background);
+	color: var(--font-white);
```

Comment:
`color` can be `inherit`, but if you use `color-scheme: dark` it will automatically color the field correctly and you can drop these.

Reply:
This still isn't fixed. `--font-white` should not exist for 2 reasons

Reply:
should be resolved now 🙂

Reply:
> `--font-white` should not exist